datasets/wegenbestand/dataset.json

```diff
@@ -17,10 +17,10 @@
   "dateCreated": "2022-02-09T00:00:00+01:00",
   "license": "Creative Commons, Naamsvermelding",
   "hasBeginning": "2022-02-09T00:00:00+01:00",
-  "accrualPeriodicity": "dagelijks",
-  "creator": "bronhouder onbekend",
-  "publisher": "Datateam Generiek",
-  "description": "Aangewezen wegen voor vrachtverkeer met gevaarlijkestoffen of valt onder de categorie zwaar verkeer.",
+  "accrualPeriodicity": "per kwartaal",
+  "creator": "@Verkeersonderzoek",
+  "publisher": "Datateam Mobiliteit",
+  "description": "Informatie over de status van wegen op het gebied van beleidsmatige status, wettelijke snelheid en andere beperkingen.",
```

Comment:
The description in the ticket ends with `beperkingen"`, not `beperkingen."`. That is, without the full stop.